core/controllers/suggestion.py

```diff
@@ -765,12 +765,13 @@ def get(self, target_type: str, suggestion_type: str) -> None:
         suggestions: Sequence[suggestion_registry.BaseSuggestion] = []
         next_offset = 0
         if suggestion_type == feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT:
-            reviewable_suggestions = []
-            if (exp_ids and len(exp_ids) > 1 and limit is None):
+            reviewable_suggestions: List[
+                suggestion_registry.SuggestionTranslateContent] = []
+            if (exp_ids and limit is None):
                 reviewable_suggestions, next_offset = (
                     suggestion_services
                     .get_reviewable_translation_suggestions_for_single_exp(
-                        self.user_id, exp_ids[0], sort_key))
+                        self.user_id, exp_ids[0]))
```

Comment:
Bit confused about this. If exp_ids is passed then how do you know it only has 1 element? If that is a necessity then shouldn't there be some kind of assertion or check here?

Reply:
Done, I have added a check to check if len(exp_id) is 1